LLama/Sampling/DefaultSamplingPipeline.cs

```diff
@@ -111,7 +111,7 @@ public float AlphaPresence
     /// <summary>
     /// Seed to use for random sampling
     /// </summary>
-    public uint Seed { get; set; } = 42;
+    public uint Seed { get; set; } = (uint) new Random().Next(0, int.MaxValue);
```

Comment:
I think it would be better to use [`RandomNumberGenerator`](https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.randomnumbergenerator?view=net-9.0) or a single static Random instance (protected by a lock). Afaik `new Random()` doesn't have very good distribution if you create many all at the same time.

Reply:
Would lean towards expecting the users to manually specify `Seed` on those cases. Thoughts?

Reply:
Even so, It's a potential footgun that's easy to fix, so we may as well!